components/FlyForm.js

```diff
@@ -19,25 +19,28 @@ class FlyForm extends Component {
   handleChange = (property, text) => {
     this.setState({ property: text });
     this.props.updateFlyEntry(property, text);
-  }
+  };
 
   render() {
     return (
       <View style={styles.container}>
+        <Text style={styles.label}>Fly Name:</Text>
```

Comment:
Thanks for adding these labels for these input fields in the fly editing and adding page.  This make things more understandable for the user when filling out fly information.

Reply:
You're welcome! Thanks for bringing this to the group's attention!